src/google/protobuf/util/json_format_proto2.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
+
+package proto2;
+
+option java_package = "com.google.protobuf.util";
+option java_outer_classname = "JsonFormatProto2";
+
+enum EnumValue {
```

Comment:
Could you add these to `json_format.proto` instead of creating a new file? I think that might simplify things since `json_format.proto` already exists and is proto2.

Reply:
done